### DISCUSSION TITLE
Votes on alternatives

### DISCUSSION THREAD

**Discussion by 111phil** [OpenSlides/OpenSlides#6367] [Ideas]
As of now, there is unfortunately not the possible to vote on alternatives in a motion; for example:

> Do you prefer the park bench to be red, white or yellow?

**Comment by ostcar** [comment]
We will not change a poll on a motion. Our motion system is complex enough as it is and we are careful when adding new features to it.

But we are adding the possibility to adding a flexible poll to each agenda item. There it will be possible to adding more then one option (like on the assignment poll where you can also add more then one candidate as option).

Maybe this is a workaround for you.
